frontend/packages/kubevirt-plugin/src/selectors/vm/flavor-text.ts

```diff
@@ -13,17 +14,24 @@ export const getFlavorText = ({
   cpu,
   memory,
   flavor,
+  t,
+  addMemoryText = true,
 }: {
   cpu: CPURaw;
   memory: string;
   flavor?: string;
+  t: TFunction;
```

Comment:
we will need a follow-up to move the `t` parameter out of here and do the actual translation in the component level, but this will fix the bug for now